fonts/Licenses/Buntype Font Software End User License Agreement.html

```diff
@@ -0,0 +1,44 @@
+<html><head>
```

Comment:
We discussed using a similar, open source font here to avoid licensing issues. That would be great to do.